fastlane/Fastfile

```diff
@@ -67,7 +67,7 @@ end
 
 lane :is_latest_video_renderer do |_options|
   result = false
-  Dir.chdir('../Carthage/Checkouts/OneMobileSDK-videorenderer-ios') do
+  Dir.chdir('../Carthage/Checkouts/VerizonVideoPartnerSDK-videorenderer-ios') do
```

Comment:
Is it even valid?

Reply:
Who knows - these scripts are no longer used and it is highly possible we will drop them.